src/ir/proto/sql/decode.cc

```diff
@@ -35,4 +36,27 @@ Value DecodeSourceTableColumn(
       decoder_context.GetOrCreateStorage(column_path)));
 }
 
+Value DecodeLiteral(
+    const Literal &literal, DecoderContext &decoder_context) {
+  const std::string &literal_str = literal.literal_str();
+  CHECK(!literal_str.empty()) << "required field literal_str was empty.";
+
+  // For now, we are abusing the `Storage` IR node to encode literals. This is
```

Comment:
Instead of using `Storage`, why don't you use a `const` operation with the string as a `StringAttribute`?

Reply:
Ah, that's a good point. I'll change the approach to do that.

Reply:
Done.